extension/src/plots/paths/model.test.ts

```diff
@@ -575,4 +578,144 @@ describe('PathsModel', () => {
     model.checkIfHasPreviousCustomSelection()
     expect(setHasCustomSelectionSpy).toHaveBeenCalledWith(true)
   })
+
+  it('should return false when calling getHasTooManyPlots when there is a custom selection', () => {
+    const model = new PathsModel(
+      mockDvcRoot,
+      buildMockErrorsModel(),
+      buildMockMemento()
+    )
+
+    model.setHasCustomSelection(true)
+
+    expect(model.getHasTooManyPlots()).toBe(false)
+  })
+
+  it('should return false when calling getHasTooManyPlots when there is no custom selection and there are fewer than 20 plots', () => {
+    const model = new PathsModel(
+      mockDvcRoot,
+      buildMockErrorsModel(),
+      buildMockMemento()
+    )
+
+    model.setHasCustomSelection(false)
+
+    const nodes = [{}]
+    for (let i = 0; i < 19; i++) {
+      nodes.push({})
+    }
```

Comment:
Minor! I think we can create `nodes` for this test and the next one in a single line: 

```js
const nodes = Array.from({ length: 20 }).fill({})
```